ArduPlane/fence.cpp

```diff
@@ -98,6 +87,21 @@ void Plane::fence_check()
                     guided_WP_loc.lat = home.lat;
                     guided_WP_loc.lng = home.lng;
                 }
+
+                float fence_alt_max, fence_alt_min;
+                if (fence.get_return_altitude() > 0) {
```

Comment:
The description for this parameter doesn't make it clear that 0 is a flag value.  As it stands someone reading the parameter description may think the vehicle will transit to 0m on default on fence-rtl.  This is a pre-existing issue.